docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -91,19 +91,40 @@ This analysis can be categorised as relating
 to the energetics of the different lambda states that were sampled,
 or to the analysis of the change in structural conformation over time in each state.
 
-* lambda energetic analysis
-
-  * free energy overlap matrix
-  * replica exchange overlap matrix (for repex only)
-  * timeseries of replica states
-  * forward and reverse estimates of free energies
-
-* relating to structural analysis. If a protein was present, these analyses first center and align the system so that
+.. todo: issue 792, consolidate this page into its own analysis page and link both RBFE and AFE pages to it
+
+* Energetic and replica exchange analysis.  These analyses consider the swapping and energetic overlap between the
+  different simulated states to try to establish the convergence and correctness of the estimate of free energy
+  difference produced.
+
+  * MBAR overlap matrix.  This plot is used to assess if the different lambda states simulated overlapped energetically.
+    Each matrix element represents the probability of a sample from a given row state being observable in a given column
+    state.
+    This plot should show that the diagonal of the matrix has some "width" so that the two end states are connected.
```

Comment:
Maybe on could specifically say that the values above the main diagonal (is that what the off-diagonal 1 is called?) should be greater or equal 0.03 (3%) (citing alchemlyb here: "It is recommended that there should be at least 0.03 [[Klimovich2015]] overlap between neighboring states." (https://alchemlyb.readthedocs.io/en/latest/visualisation.html)